packages/yourloops/pages/login/login-page.tsx

```diff
@@ -25,101 +25,223 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import React, { FunctionComponent } from 'react'
+import React, { FunctionComponent, useEffect } from 'react'
 import { useTranslation } from 'react-i18next'
 import { useAuth0 } from '@auth0/auth0-react'
 
 import { Theme } from '@mui/material/styles'
 import { makeStyles } from 'tss-react/mui'
-import Button from '@mui/material/Button'
-import Card from '@mui/material/Card'
-import CardActions from '@mui/material/CardActions'
-import CardContent from '@mui/material/CardContent'
-import CardMedia from '@mui/material/CardMedia'
-import Container from '@mui/material/Container'
-import Typography from '@mui/material/Typography'
+import loginPageBackground from 'login-page-background.png'
+import loginPageLaptop from 'login-page-laptop.png'
 
-import appConfig from '../../lib/config/config'
+import Toolbar from '@mui/material/Toolbar'
+import AppBar from '@mui/material/AppBar'
+import config from '../../lib/config/config'
+import { Link as RouterLink, useHistory } from 'react-router-dom'
+import Box from '@mui/material/Box'
+import Link from '@mui/material/Link'
+import LanguageSelect from '../../components/language-select'
+import LanguageIcon from '@mui/icons-material/Language'
+import Typography from '@mui/material/Typography'
+import { GlobalStyles } from 'tss-react'
+import { useAlert } from '../../components/utils/snackbar'
+import Button from '@mui/material/Button'
 
-const loginStyle = makeStyles({ name: 'login-page-styles' })((theme: Theme) => {
-  return {
-    card: {
-      padding: theme.spacing(2)
-    },
-    cardContent: {
-      textAlign: 'center',
-      margin: `${theme.spacing(2)} 0`
-    },
-    cardActions: {
-      justifyContent: 'center'
+const LOGO_COLOR_LIGHT = '#40BAE9'
+const styles = makeStyles({ name: 'login-page-styles' })((theme: Theme) => ({
+  appBar: {
+    borderBottom: `1px solid ${theme.palette.divider}`,
+    zIndex: theme.zIndex.drawer + 1,
+    backgroundColor: theme.palette.common.white,
+    color: 'var(--text-base-color)',
+    paddingBlock: theme.spacing(2),
+    paddingInline: theme.spacing(7)
+  },
+  backgroundImage: {
+    position: 'absolute',
+    right: 0,
+    bottom: -1,
+    zIndex: -1
+  },
+  button: {
+    marginRight: theme.spacing(2),
+    paddingInline: theme.spacing(3),
+    textTransform: 'capitalize',
+    width: 136
+  },
+  heading: {
+    fontFamily: 'MuseoSlab',
+    fontSize: '1.5rem',
```

Comment:
I don't think so for this one because Brandcom gave me some instructions and values that were not created with MUI standards... for example 1.1rem size is not listed in MUI theme 🤷🏻‍♂️